paddle/fluid/framework/variable.h

```diff
@@ -27,12 +27,12 @@ class Variable {
  public:
   template <typename T>
   const T& Get() const {
-    static_assert(
-        IsRegisteredVarType<T>(),
-        "Not registered type. Please register T inside var_type_traits.h");
-    PADDLE_ENFORCE(holder_ != nullptr, "Variable must hold some thing");
+    static_assert(IsRegisteredVarType<T>(),
```

Comment:
`static_assert` never throws to users. It is not necessary to change this line.

Reply:
Done